src/eradiate_plugins/integrators/piecewise_volpath.cpp

```diff
@@ -25,68 +24,73 @@ Piecewise volumetric path tracer (:monosp:`volpath`)
+   - Specifies the minimum path depth, after which the implementation will start
+to use the *russian roulette* path termination criterion. (Default: 5)
 
  * - hide_emitters
    - |bool|
    - Hide directly visible emitters. (Default: no, i.e. |false|)
 
-This plugin provides a piecewise volumetric path tracer that can be used to compute approximate 
-solutions of the radiative transfer equation for plane parallel geometry. Its implementation 
-makes use of multiple importance sampling to combine BSDF and phase function sampling with 
-direct illumination sampling strategies. On surfaces, it behaves exactly like the standard path tracer. 
+This plugin provides a piecewise volumetric path tracer that can be used to
+compute approximate solutions of the radiative transfer equation for plane
+parallel geometry. Its implementation makes use of multiple importance sampling
+to combine BSDF and phase function sampling with direct illumination sampling
+strategies. On surfaces, it behaves exactly like the standard path tracer.
 
-This integrator only works with piecewise media. It uses assumptions on the medium's geometry to 
-accelerate the sampling and transmission calculations. 
+This integrator only works with piecewise media. It uses assumptions on the
+medium's geometry to accelerate the sampling and transmission calculations.
 
-This integrator has special support for index-matched transmission events (i.e. surface scattering
-events that do not change the direction of light). As a consequence, participating media enclosed by
-a stencil shape are rendered considerably more efficiently when this shape
-has a :ref:`null <bsdf-null>` or :ref:`thin dielectric <bsdf-thindielectric>` BSDF assigned
-to it (as compared to, say, a :ref:`dielectric <bsdf-dielectric>` or
-:ref:`roughdielectric <bsdf-roughdielectric>` BSDF).
+This integrator has special support for index-matched transmission events (i.e.
+surface scattering events that do not change the direction of light). As a
+consequence, participating media enclosed by a stencil shape are rendered
+considerably more efficiently when this shape has a :ref:`null <bsdf-null>` or
+:ref:`thin dielectric <bsdf-thindielectric>` BSDF assigned to it (as compared
+to, say, a :ref:`dielectric <bsdf-dielectric>` or :ref:`roughdielectric
+<bsdf-roughdielectric>` BSDF).
 
 .. note:: This integrator does not implement good sampling strategies to render
-    participating media with a spectrally varying extinction coefficient. For these cases,
-    it is better to use the more advanced :ref:`volumetric path tracer with
-    spectral MIS <integrator-volpathmis>`, which will produce in a significantly less noisy
-    rendered image.
+    participating media with a spectrally varying extinction coefficient. For
+these cases, it is better to use the more advanced :ref:`volumetric path tracer
+with spectral MIS <integrator-volpathmis>`, which will produce in a
+significantly less noisy rendered image.
```

Comment:
This is where clang-format is a bit idiotic: it breaks significant indentation in comments :( Can you fix that?